setup.cfg

```diff
@@ -3,7 +3,7 @@ name = columbo
 url = https://github.com/plannigan/columbo
 author = Patrick Lannigan
 author_email = p.lannigan@gmail.com
-version = 0.13.0.post1
+version = 0.13.0
```

Comment:
`hyper-bump-it` doesn't support the `post` version. So, remove it in preparation for the next release. (The existing bump2version configuration also didn't support it)